src/Darnytsia.Creatio.Core/Features/Contacts/Commands/UpdateContactBirthDayCommand.cs

```diff
@@ -0,0 +1,5 @@
+﻿using System;
+
+namespace Darnytsia.Creatio.Core.Features.Contacts.Commands;
+
+public record UpdateContactBirthDayCommand(Guid ContactId, DateTime BDay) : IRequest;
```

Comment:
```suggestion
public record UpdateContactBirthDayCommand(Guid ContactId, DateTime Birthday) : IRequest;
```
try not to use short form of words